src/ngraph/op/fused/leaky_relu.cpp

```diff
@@ -0,0 +1,48 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/op/fused/leaky_relu.hpp"
+#include "ngraph/node.hpp"
+#include "ngraph/op/maximum.hpp"
+#include "ngraph/op/multiply.hpp"
+#include "ngraph/op/util/broadcasting.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+op::LeakyRelu::LeakyRelu(const shared_ptr<Node>& data, const shared_ptr<Node>& alpha)
+    : FusedOp("LeakyRelu", {data, alpha})
+{
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::LeakyRelu::decompose_op() const
+{
+    auto data = get_argument(0);
+    auto alpha_node = get_argument(1);
+
+    alpha_node = ngraph::op::make_broadcast_node(alpha_node, data->get_shape());
```

Comment:
Broadcasting `[2, 1]` with `[1, 3]` will fail with this broadcasting.
Consider use of numpy style broadcasting.